hbase-server/src/test/java/org/apache/hadoop/hbase/protobuf/TestProtobufUtil.java

```diff
@@ -348,4 +352,25 @@ public void testToCell() throws Exception {
         ProtobufUtil.toCell(ExtendedCellBuilderFactory.create(CellBuilderType.SHALLOW_COPY), cell);
     assertTrue(CellComparatorImpl.COMPARATOR.compare(offheapKV, newOffheapKV) == 0);
   }
+
+  @Test
+  public void testMetaRegionState() throws Exception {
```

Comment:
This covers the "new" state info, but what about the old serialized version?

Reply:
Done.